### ISSUE TITLE
bash autocompletion leaves a floating "Succeeded" command in `bash_completion.d/kapp`

### ISSUE THREAD

**Issue by jamespollard8** [k14s/kapp#175]
**What steps did you take:**
Update to kapp v0.35.
Run `kapp completion bash > $(brew --prefix)/etc/bash_completion.d/kapp`
`source` my bash completion

**What happened:**
```
-bash: Succeeded: command not found
```

**Anything else you would like to add:**
Looks like there's simply an errant "Succeeded" at the end of the `bash_completion.d/kapp` file:

```
$ tail -n3 $(brew --prefix)/etc/bash_completion.d/kapp

# ex: ts=4 sw=4 et filetype=sh
Succeeded
```

**Comment by cppforlife** [comment]
darn, it's because we are setting --tty=true by default (instead of detecting it; choice made early on in kapp). as a workaround you can add --tty=false to command above. need to address properly of course.

**Comment by cppforlife** [comment]
related: https://github.com/k14s/kapp/issues/28 -- https://github.com/vmware-tanzu/carvel-kapp/commit/a389427338965db48bf0c21f1383f8ecf115da20#diff-bb0f7dd7ddc2dc53d56a9f5ba035b6f9c53fe97faa3f3a9f9aa7e3d336081058R19-R20

**Comment by cari-lynn** [comment]
This fix is available in release [v0.36.0](https://github.com/vmware-tanzu/carvel-kapp/releases/tag/v0.36.0)
